data/ysoserial_payloads.json

```diff
@@ -317,6 +317,17 @@
     },
     "Wicket1": {
       "status": "unsupported"
+    },
+    "frohoff/ysoserial#168": {
```

Comment:
Named to point to https://github.com/frohoff/ysoserial/pull/168 so both PRs can move forward independently of each other.